src/ht4o/Logging.cs

```diff
@@ -230,7 +230,6 @@ private static TraceSource GetTraceSource()
 #if DEBUG
                 var listeners = new List<TraceListener>();
                 listeners.AddRange(Trace.Listeners.OfType<TraceListener>());
-                listeners.AddRange(Debug.Listeners.OfType<TraceListener>());
```

Comment:
In .net standard the Debug and Trace share the collection of the listeners. 
https://stackoverflow.com/a/53847617/996233